app/builders/content_change_email_builder.rb

```diff
@@ -41,42 +41,34 @@ def subject(content_change)
   end
 
   def body(content_change, subscription, address)
+    list = subscription.subscriber_list
+
     <<~BODY
       #{I18n.t!('emails.content_change.opening_line')}
 
+      # #{list.title}
+
       ---
-      #{presented_content_change(content_change, subscription)}
+
+      #{middle_section(list, content_change)}
+
       ---
-      #{footer(subscription, address).strip}
-    BODY
-  end
 
-  def presented_content_change(content_change, subscription)
-    copy = ContentChangePresenter.call(content_change)
+      # #{I18n.t!('emails.content_change.footer_header')}
 
-    subscriber_list = subscription.subscriber_list
-    if subscriber_list.description.present?
-      copy += "\n#{subscriber_list.description}\n"
-    end
+      #{I18n.t!('emails.content_change.footer_explanation')}
 
-    copy
-  end
+      #{list.title}
 
-  def footer(subscription, address)
-    <<~BODY
-      #{permission_reminder(subscription.subscriber_list)}
+      # [Unsubscribe](#{PublicUrls.unsubscribe(subscription)})
 
-      #{ManageSubscriptionsLinkPresenter.call(address)}
+      [#{I18n.t!('emails.content_change.footer_manage')}](#{PublicUrls.authenticate_url(address: address)})
     BODY
   end
 
-  def permission_reminder(subscriber_list)
-    topic = if subscriber_list.url
-              "[#{subscriber_list.title}](#{Plek.new.website_root}#{subscriber_list.url})"
-            else
-              subscriber_list.title
-            end
-
-    I18n.t!("emails.content_change.permission_reminder", topic: topic)
+  def middle_section(list, content_change)
```

Comment:
Minor, I found this method name a bit surprising as it didn't feel very descriptive. I can see problems with the previous `presented_content_change` name too though and don't have any better suggestions so don't consider this a blocking comment.

Reply:
Yeah, it was annoying to have to add it, due to the optional description field. I can't think of a significantly clearer name either, so I'll leave it as-is.